cache/in-memory/src/updates.rs

```diff
@@ -1193,10 +1191,10 @@ mod tests {
             let entry = cache.0.users.get(&UserId(3)).unwrap();
             assert_eq!(entry.value().1.len(), 1);
         }
-        assert_eq!(
-            cache.member(GuildId(1), UserId(3)).unwrap().user_id,
-            UserId(3),
-        );
+        // assert_eq!(
```

Comment:
Meant to be commented out or an artifact of testing changes?

Reply:
Good catch